translate/convert/test_po2ts.py

```diff
@@ -77,8 +77,7 @@ def test_duplicates(self):
 msgstr "b"
 '''
         tsfile = self.po2ts(posource)
-        print(tsfile)
```

Comment:
i assumed that these were debugging

but, yep, i should have put this cleanup in a separate commit